client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -3,22 +3,103 @@
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { withLocalizedMoment } from 'components/localized-moment';
+import SecurityIcon from 'landing/jetpack-cloud/components/security-icon';
+
+import './style.scss';
 
 class ScanPage extends Component {
+	renderScanOkay() {
+		const { siteSlug, moment, lastScanTimestamp, nextScanTimestamp } = this.props;
+
+		let nextScanString = translate( 'tomorrow' );
+		if ( moment( nextScanTimestamp ).isSame( Date.now(), 'd' ) ) {
+			nextScanString = translate( 'later today' );
+		}
+
+		return (
+			<>
+				<SecurityIcon className="scan__icon" />
+				<h1 className="scan__header scan__header--okay">
+					{ translate( 'Don’t worry about a thing' ) }
+				</h1>
+				<p>
+					{ translate(
```

Comment:
Lets not translate this part just yet. 
Since I am not 100% sure we will be including a way to run manual scans with V1.

Reply:
Removed in ee35e139de69f85744dbb761641e083cdfcaebdb.